CronofyCSharpSampleApp/Views/Availability/Index.cshtml

```diff
@@ -87,4 +88,5 @@
         </tr>
         }
     </tbody>
-</table>
\ No newline at end of file
+</table>
+}
```

Comment:
Is this on initial load or when there are no AvailablePeriods after we've looked? if its the latter we should show an error